packages/core/src/client.ts

```diff
@@ -87,6 +87,8 @@ export abstract class Client {
 
   protected abstract checkIn(id: string): Promise<void>
 
+  protected abstract showUserFeedbackForm(options: UserFeedbackForOptions): Promise<void>
```

Comment:
I used the same approach we have for `checkIn`: Feature is supported only one on client side, so we throw an error if used on the server side.